extensions/xkcd/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "xkcd",
+  "title": "xkcd",
```

Not sure about naming, but would be better to provide something more specific here and Capitalised. Like **XKCD Comics**
```suggestion
  "title": "XKCD Comics",
```